src/content/app/species-selector/components/species-search-field/SpeciesSearchField.scss

```diff
@@ -2,6 +2,14 @@
 
 $rightCornerWidth: 40px;
 
+label {
+  color: $dark-grey;
+  display: block;
+  font-size: 12px;
+  margin-left: 25px;
+  margin-bottom: 10px;
+}
```

Comment:
Using a tag name in a CSS module won't have it scoped to this component, which means that by writing this you are affecting global styles. Not good.

![image](https://user-images.githubusercontent.com/6834224/201079835-67d14571-c4a1-4045-bf2c-1fc949dc2d45.png)

Reply:
✅